comp/core/autodiscovery/listeners/listeners.go

```diff
@@ -7,7 +7,7 @@
 package listeners
 
 const (
-	cloudFoundryBBSListenerName = "cloudfoundry_bbs"
+	cloudFoundryBBSListenerName = "cloudfoundry-bbs"
```

Comment:
I think we should support both names to maintain compatibility.
Otherwise, this will break for users who have already adopted the new name, since it’s been there for several releases.

Reply:
@davidor  That's a good point, but not in the case of Cloud Foundry because users install the DCA through our own packaged BOSH release. And In that package, we hardcode the listener-name's value:

https://github.com/DataDog/datadog-cluster-agent-boshrelease/blob/master/jobs/datadog-cluster-agent/templates/datadog-cluster.yaml.erb#L50-L51.

We're gonna change this value temporarily to make a bugfix release and revert it to this "original" name by the next agent release.

https://github.com/DataDog/datadog-cluster-agent-boshrelease/compare/master...noueman/fix-cloud-foundry-listener-name

Reply:
Thanks for clarifying. I'm not familiar with how the cloudfoundry integration works. If that's the only way of using it, it should be fine :+1:

Reply:
Thanks @davidor, I will try to clarify these information in the description in the future.